Mathlib/Algebra/Homology/ShortComplex/ModuleCat.lean

```diff
@@ -147,6 +147,22 @@ lemma toCycles_moduleCatCyclesIso_hom :
     Category.assoc, S.moduleCatCyclesIso_hom_subtype, toCycles_i]
   rfl
 
+theorem mapCyclesIso_trans_forget₂_moduleCatCyclesIso (M : ShortComplex (ModuleCat R)) :
+    (M.mapCyclesIso (forget₂ (ModuleCat R) Ab)) ≪≫
+      (forget₂ (ModuleCat R) Ab).mapIso M.moduleCatCyclesIso = ShortComplex.abCyclesIso _ := by
+  refine Iso.ext <| (Iso.inv_eq_inv _ _).1 <|
+    (cancel_mono (M.map (forget₂ (ModuleCat R) Ab)).iCycles).1 ?_
+  simpa only [Iso.trans_inv, Functor.mapIso_inv, ← Functor.map_comp, Category.assoc, mapCyclesIso,
+    LeftHomologyData.cyclesIso_inv_comp_iCycles, abCyclesIso]
+    using congr((forget₂ (ModuleCat R) Ab).map $(moduleCatCyclesIso_inv_iCycles _))
```

Comment:
The following looks more readable to me:
```suggestion
theorem mapCyclesIso_trans_forget₂_moduleCatCyclesIso (M : ShortComplex (ModuleCat R)) :
    (M.mapCyclesIso (forget₂ (ModuleCat R) Ab)) ≪≫
      (forget₂ (ModuleCat R) Ab).mapIso M.moduleCatCyclesIso = ShortComplex.abCyclesIso _ := by
  ext : 1
  rw [← Iso.inv_eq_inv, ← cancel_mono (M.map (forget₂ (ModuleCat R) Ab)).iCycles]
  simpa only [Iso.trans_inv, Functor.mapIso_inv, ← Functor.map_comp, Category.assoc, mapCyclesIso,
    LeftHomologyData.cyclesIso_inv_comp_iCycles, abCyclesIso]
    using ((forget₂ (ModuleCat R) Ab).congr_map (moduleCatCyclesIso_inv_iCycles _))
```
I think the lemmas for `inv` and `hom` of the composition should be stated separately (and the `Iso.trans` version deduced from the `inv` lemma).